java/test/jmri/jmrit/display/layoutEditor/LayoutEditorToolsTest.java

```diff
@@ -604,45 +603,47 @@ public void setUp() throws Exception {
                 String uBlockName = "Block " + i;
                 InstanceManager.getDefault(LayoutBlockManager.class).createNewLayoutBlock(sBlockName, uBlockName);
             }
-            layoutBlocks = InstanceManager.getDefault(LayoutBlockManager.class).getNamedBeanList();
+            layoutBlocks = InstanceManager.getDefault(LayoutBlockManager.class).getNamedBeanSet().stream().collect(Collectors.toList());
```

Comment:
Why use Collectors and not `layoutBlocks = new ArrayList<>(InstanceManager.getDefault(LayoutBlockManager.class).getNamedBeanSet());`?

(Not having tried this myself, it seems that might be more readable.)

Reply:
Even better, change `layoutBloxks` declaration to a `Set` and use it natively.

Reply:
Other parts of the test are requesting specific items by indexes in order (sometimes repeating the same index), which is why I didn't suggest using a `Set`.

Reply:
> Why use Collectors and not `layoutBlocks = new ArrayList<>(InstanceManager.getDefault(LayoutBlockManager.class).getNamedBeanSet());`?
> 
> (Not having tried this myself, it seems that might be more readable.)

I actually like how streaming to the collector reads.

Reply:
> Even better, change `layoutBloxks` declaration to a `Set` and use it natively.

As Randall said, we are pulling specific elements and I am trying to change the actual tests as little as possible.

That said, this test class was initially written with arrays for all of the named bean objects, with each array initialized in a common loop.  I broke the loop into 4 parts.  I don’t think most of the registered objects are actually used, so it may be a good idea to reduce the number of objects created to only those actually used.

I’ll take that bit of refactoring on next, but I want to see if explicitly closing the layout editors helps on Jenkins first.  Neither of the jenkins jobs that run ‘ant logalltest’  have been completing successfully for a while now.